### PR TITLE
Codeofconduct

### PR DESCRIPTION
**Pull request by ashishmangla0**
image

### CODE REVIEW

File: css/pages/coc.css [hunk 1]
```diff
@@ -1,34 +1,337 @@
...
+  margin-top: 0;
+}
+.block__content p {
+  font-size: 1.125rem;
+}
+.block--bg-gray {
+  background: #f2f2f2;
+}
+
+@keyframes moveLeftBounce {
```
Comment (strider99) [inline css/pages/coc.css:194]: @ashishmangla0  do we need the animation

**Review by strider99 — COMMENTED** [review]
css looks good

**Comment by ashishmangla0** [comment]
Thanks

File: discord.html [hunk 2]
```diff
@@ -22,16 +22,27 @@
...
+        <div class="header__right">
           <a href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
-            class="btn btn--border btn--border-primary">Login</a>
+            class="btn btn--border btn--border-primary header__login-btn">
+            Login
+          </a>
+          <button class="btn header__menu-toggle">
+            <span></span>
+            <span></span>
+            <span></span>
```
Comment (ankushdharkar) [inline discord.html:44]: Why 3 empty spans?

File: code-of-conduct.html [hunk 3]
```diff
@@ -1,43 +1,125 @@
...
+								<li class="coc-list-elements">No trolling of members on the group. We are here to learn and help
+									each other grow. </li>
+								<li class="coc-list-elements">No spamming off topic stuff. Time is the most important resource. Lets
+									save it in manifolds by reducing off topic content. </li>
+								<li class="coc-list-elements">Avoid Direct Messages. Do not hesitate to ask your queries on the
+									#support discord channel. Aids in quicker response. </li>
+								<li class="coc-list-elements">Avoid using tags @everyone and @here until it is very important.</li>
+								<li class="coc-list-elements">Google your queries before you seek help. </li>
+								<li class="coc-list-elements">When seeking help, submit the code in an easy-to-evaluate fashion.
+									Share a <a href="https://codepen.io/" target="_blank" rel="nopener norefer">Codepen </a>link
```
Comment (ankushdharkar) [inline code-of-conduct.html:101]: Type. Correct: `noopener noreferrer`

File: code-of-conduct.html [hunk 4]
```diff
@@ -1,43 +1,125 @@
...
+					</div>
+
+				</div>
+				
+			</div>
+		</section>
+	</main>
+	<footer>
+		<p class="info-repo">The contents of this website are deployed from this <a
+				href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank" rel="nopener norefer"> open
```
Comment (ankushdharkar) [inline code-of-conduct.html:120]: Please fix: `noopener noreferrer`

File: code-of-conduct.html [hunk 5]
```diff
@@ -1,43 +1,125 @@
...
+									each other grow. </li>
+								<li class="coc-list-elements">No spamming off topic stuff. Time is the most important resource. Lets
+									save it in manifolds by reducing off topic content. </li>
+								<li class="coc-list-elements">Avoid Direct Messages. Do not hesitate to ask your queries on the
+									#support discord channel. Aids in quicker response. </li>
+								<li class="coc-list-elements">Avoid using tags @everyone and @here until it is very important.</li>
+								<li class="coc-list-elements">Google your queries before you seek help. </li>
+								<li class="coc-list-elements">When seeking help, submit the code in an easy-to-evaluate fashion.
+									Share a <a href="https://codepen.io/" target="_blank" rel="nopener norefer">Codepen </a>link
+									instead of a <a href="https://github.com/" target="_blank" rel="nopener norefer">Github
```
Comment (ankushdharkar) [inline code-of-conduct.html:102]: Please fix: `noopener noreferrer`

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: discord.html [hunk 6]
```diff
@@ -22,16 +22,27 @@
           </a>
         </div>
         <div class="header__center">
+          <span class="header__mobile-toggle">
+            <span></span>
```
Comment (ankushdharkar) [inline discord.html:26]: Why empty spans?

File: discord.html (hunk 2 above)
Comment (ashishmangla0) [reply discord.html:44]: Span tags are bars for responsive.

File: code-of-conduct.html (hunk 4 above)
Comment (ashishmangla0) [reply code-of-conduct.html:120]: I will also fix these

File: discord.html (hunk 2 above)
Comment (ankushdharkar) [reply discord.html:44]: Please use classes to denote this

File: code-of-conduct.html (hunk 4 above)
Comment (ashishmangla0) [reply code-of-conduct.html:120]: fixed noopener noreferrer

File: discord.html (hunk 6 above)
Comment (ashishmangla0) [reply discord.html:26]: fixed

File: discord.html (hunk 2 above)
Comment (ashishmangla0) [reply discord.html:44]: its fixed

File: code-of-conduct.html (hunk 3 above)
Comment (ashishmangla0) [reply code-of-conduct.html]: fixed that

File: code-of-conduct.html (hunk 5 above)
Comment (ashishmangla0) [reply code-of-conduct.html]: fixed

**Pull request closed without merging**